org.widelands.widelands.json

```diff
@@ -0,0 +1,478 @@
+                "--with-x",
+                "--with-threads",
+                "--with-magick_plus_plus",
+                "--with-gslib",
+                "--with-wmf",
+                "--with-webp",
+                "--with-openexr",
+                "--with-rsvg",
+                "--with-xml",
+                "--with-jbig",
+                "--with-openjp2",
+                "--prefix=/app/opt/imagemagick",
+                "--exec-prefix=/app/opt/imagemagick"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.imagemagick.org/download/ImageMagick-6.9.9-44.tar.xz",
+                    "sha256": "4de6340f22563d80b93ee22b799c023d3a816d59be0b2fea8dd6ebc7b8fd4c7f"
+                },
+                {
+                    "type": "patch",
+                    "path": "ImageMagick-6.9.9-3-multiarch-implicit-pkgconfig-dir.patch",
+                    "strip-components": 1,
+                    "sha256": "80eea73e19a47cdb322d9006d4fe4a868db0d0d43fd7a0c9ae1a5e2fbbdbfe4b"
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "echo 'copy examples';",
+                        "mkdir 'Magick++/examples'; cp -p Magick++/demo/*.cpp Magick++/demo/*.miff Magick++/examples;"
+                    ]
+                }
+            ],
+            "post-install": [
+                "echo 'post-install: docs';",
+                "install -d '/app/opt/imagemagick/share/doc/imagemagick'; install -p -m 0644 'AUTHORS.txt' 'ChangeLog' 'Install-mac.txt' 'Install-unix.txt' 'Install-vms.txt' 'Install-windows.txt' 'NEWS.txt' 'NOTICE' 'Platforms.txt' 'README.txt' 'QuickStart.txt' '/app/opt/imagemagick/share/doc/imagemagick/';",
+                "install -d '/app/opt/imagemagick/share/doc/imagemagick/Magick++'; install -p -m 0644 'Magick++/AUTHORS' 'Magick++/ChangeLog' 'Magick++/NEWS' 'Magick++/README' '/app/opt/imagemagick/share/doc/imagemagick/Magick++/';",
+                "install -d '/app/opt/imagemagick/share/doc/imagemagick/Magick++/examples'; install -p -m 0644 Magick++/examples/* '/app/opt/imagemagick/share/doc/imagemagick/Magick++/examples/';",
+                "echo 'post-install: licenses';",
+                "install -d '/app/opt/imagemagick/share/licenses/imagemagick'; install -p -m 0644 'LICENSE' '/app/opt/imagemagick/share/licenses/imagemagick/';",
+                "install -d '/app/opt/imagemagick/share/licenses/imagemagick/Magick++'; install -p -m 0644 'www/Magick++/COPYING' '/app/opt/imagemagick/share/licenses/imagemagick/Magick++/';"
+            ],
+            "cleanup": [
+                "/opt/imagemagick/share/doc",
+                "/opt/imagemagick/share/man"
+            ]
+        },
+        "shared-modules/lua5.3/lua-5.3.2.json",
+        "shared-modules/glu/glu-9.0.0.json",
+        "shared-modules/glew/glew.json",
+        {
+            "name": "boost",
```

Comment:
Use the `simple` build system.

Reply:
I borrowed this from `org.seul.pingus.json`:
https://github.com/flathub/org.seul.pingus/blob/6a5a568ccdfe257fc8d4e663fe8e50aff9e0fe30/org.seul.pingus.json#L34-L53

Have you accidentally approved the incorrect manifest?

Reply:
It has been added over a year ago. It simply wasn't available back then.